lms/static/sass/shared/_header.scss

```diff
@@ -145,6 +145,30 @@ header.global {
           }
         }
       }
+
+      .wrapper-more-actions {
+
+        // Some overrides and customization for the button
+        // in this particular use case
+        .button-more {
+          @include margin-left(-1px); // Don't want a double border on the side with the profile name
+          @include border-top-left-radius(0);
+          @include border-bottom-left-radius(0);
+          border-color: $gray-l3;
+          height: 34px;
```

Comment:
In this case these values are needed. Unsetting them doesn't let this control align nicely with the preceeding button/link.